models/classes/pack/ItemPack.php

```diff
@@ -128,40 +130,50 @@ public function getData()
      * Set item's assets of a given type to the pack.
      *
      * @param string $type the assets type, one of those who are supported.
-     * @param string[] $assets the list of assets' URL to load
-     *
-     * @param \tao_models_classes_service_StorageDirectory $publicDirectory
+     * @param array $assets the list of assets' URL to load
      *
      * @throw InvalidArgumentException
      */
-    public function setAssets($type, $assets, $publicDirectory)
+    public function setAssets(string $type, $assets): void
```

Comment:
This method is used by: [QtiItemPacker](https://github.com/oat-sa/extension-tao-itemqti/blob/0612d5a9617a0fdfeea6d48ed55f0d7924ab7c3b/model/pack/QtiItemPacker.php)

Reply:
it is not used here, having an extra parameters does not throw fatal error.
It is not a good practice it is why it is removed in https://github.com/oat-sa/extension-tao-itemqti/pull/1481